debian/changelog

```diff
@@ -1,8 +1,9 @@
-kano-apps (4.2.1-0) unstable; urgency=low
+kano-apps (4.3.1-0) unstable; urgency=low
```

Comment:
Let's add a new entry altogether rather than editing the old one

Reply:
Done